coopr-docs/docs/source/rest/node-usage.rst

```diff
@@ -0,0 +1,174 @@
+..
+   Copyright © 2012-2015 Cask Data, Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+ 
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+:orphan:
+
+.. index::
+   single: Metrics 
+
+=======
+Metrics
+=======
+
+.. include:: /rest/rest-links.rst
+
+Admins are able to get metrics, such as the number of tasks that are in progress and queued as well as node hour usage.
+
+.. _metrics-node-usage:
+
+
+Get Node Usage Metrics
+======================
+
+To get node usage metrics, HTTP GET request to URI:
+::
+
+/metrics/nodes/usage
+
+Only admins are allowed to get node usage metrics. Tenant admins will see metrics for their own nodes,
+whereas the superadmin will get metrics for all tenant nodes.
+
+
+HTTP Responses
+==============
+
+The response will be a JSON Object with start and end time as keys, and the node usage metrics as the values.
+node usage metrics include the key value pairs entered as parameters, followed by the resulting node hour usage
+based on the selected criteria.
+
+.. list-table::
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Status Code
+     - Description
+   * - 200 (OK)
+     - If update was successful
+   * - 403 (FORBIDDEN)
+     - If the user is forbidden from getting node usage metrics.
+
+
+--------------
```

Comment:
Rather than have these horizontal lines, I would re-organize the section titles so as to not require them:
```
=======
Metrics
=======

Overview
========
Admins are able to retrieve metrics, such as the number of tasks that are in progress and queued, and the node hour usage.

Task Queue Metrics
==================
To retrieve task queue metrics, send an HTTP GET request to the URI::
...

Node Usage Metrics
==================
To retrieve node usage metrics, send an HTTP GET request to the URI::
...
HTTP Responses
-----------------------
...
API Endpoints and Examples
---------------------------------------

.. rubric:: General Usage
.. rubric:: Start Time Only
.. rubric:: End Time Only
.. rubric:: Time Range (Start and End Times)
.. rubric:: Specifying a User
.. rubric:: Specifying a Tenant

```